solution.txt

```diff
@@ -0,0 +1,205 @@
+/*
+Exercise 1:
+  - Define an empty array named foods
+*/
+
+// Exercise 1 has been completed for you...
+
+const foods = [];
+
+console.log('Exercise 1 Result:\n', foods);
+
+/*
+Exercise 2:
+  - Add the strings 'pizza' & 'cheeseburger' to the foods array such that 'pizza' comes before 'cheeseburger'.
+*/
+
+// Complete Exercise 2 below...
+foods.push('pizza')
+foods.push('cheeseburger');
+
+console.log('Exercise 2 Result:\n',  foods);
+
+/*
+Exercise 3:
+  - Add the string 'taco' to the foods array so that 'taco' is the first food in the array.
+*/
+
+// Complete Exercise 3 below...
+
+foods.unshift('taco')
+
+console.log('Exercise 3 Result:\n', foods);
+
+/*
+Exercise 4:
+  - Access the string 'pizza' (based upon its known position) in the foods array and assign to a variable named favFood.
+*/
+
+// Complete Exercise 4 below...
+
+var favFood = foods[1];
+
+console.log('Exercise 4 Result:\n', favFood);
+
+/*
+Exercise 5:
+  - Insert the string 'tofu' in the foods array between 'pizza' & 'cheeseburger'
+*/
+
+// Complete Exercise 5 below...
+
+foods.splice(foods.indexOf('pizza')+1,0,'tofu')
+
+console.log('Exercise 5 Result:\n', foods);
+
+/*
+Exercise 6:
+  - Replace the string 'pizza' in the foods array with the strings 'sushi' & 'cupcake'.
+*/
+
+// Complete Exercise 6 below...
+
+foods.splice(foods.indexOf('pizza'),1,'sushi','cupcake');
```

Comment:
nice use of a couple of array methods here! you've done it programmatically, so no matter where `pizza` is in the array, you'll always splice it out 👍